shell/deepdive-initdb

```diff
@@ -16,8 +16,19 @@ db-init "$@"
 
 # make sure the necessary tables are all created
 if [[ -e app.ddlog ]]; then
-    # TODO export schema.sql from ddlog instead of running initdb pipeline
-    deepdive-run initdb
+    if [[ $# -gt 0 ]]; then
```

Comment:
Rather than having these argument count checks buried deep inside, I think it's much clearer to define initdb's behavior entirely differently when arguments are specified.  Please see my comment on the PR for reorganizing.